core/src/main/java/com/scalar/db/storage/jdbc/JdbcDatabase.java

```diff
@@ -197,6 +197,9 @@ public void mutate(List<? extends Mutation> mutations) throws ExecutionException
       close(connection);
       throw new ExecutionException(
           CoreError.JDBC_ERROR_OCCURRED_IN_MUTATION.buildMessage(e.getMessage()), e);
+    } catch (Exception e) {
```

Comment:
@brfrn169 I took care of this similar potential leak in https://github.com/scalar-labs/scalardb/pull/2766/commits/f18d40f6765a0a61b6e73c521d6967d0551ca830.